packages/components/src/tooltip/index.js

```diff
@@ -117,7 +120,13 @@ class Tooltip extends Component {
 						aria-hidden="true"
 						animate={ false }
 					>
-						{ text }
+						<KeyboardShortcuts
```

Comment:
I'm surprised this works, if it does. `text` is the text of the tooltip itself, which I wouldn't expect should ever have focus such that the `KeyboardShortcuts` could capture keyboard events. Instead, I would expect the `KeyboardShortcuts` should wrap the `child.props.children` (the original children).

Reply:
@aduth, please let me know if the implementation is ok, so I can start adjusting failing tests. Thank you.